rollup.config.js

```diff
@@ -70,7 +75,12 @@ export default [
       }),
       nodeResolve(),
       terser({
-        compress: true,
+        compress: {
+          pure_getters: true,
+          passes: 3,
+          drop_console: true,
```

Comment:
Question: should we change only this value when we really want to debug the player with a logging?

Reply:
@hermet this config aims production build. When we want to debug, we can disable and check the log